### PR TITLE
add area, beds, checkin/out information, amenities, roomAmenities, ro…

### PR DESCRIPTION
**Pull request by mikimartin88**
…omViews, mandatoryFees

### CODE REVIEW

File: hotelx/hotstc/objects/object_TimeRange.graphql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+type TimeRange {
+  # the start time of range
+  startTime: String!
```
Comment (arquio) [inline hotelx/hotstc/objects/object_TimeRange.graphql]: DateTime?

File: hotelx/hotstc/objects/object_PropertyType.graphql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+type PropertyType{
+  # PropertyType code
+  code: String
```
Comment (arquio) [inline hotelx/hotstc/objects/object_PropertyType.graphql]: not IDs?

File: hotelx/hotstc/objects/object_Location.graphql [hunk 3]
```diff
@@ -23,4 +23,7 @@ type Location {
 
   # Detailed information about the destinations geonames
   geoNames:[Geoname!]!
+
+  # the state/province of the property
+  state: String
```
Comment (arquio) [inline hotelx/hotstc/objects/object_Location.graphql]: better use scalar?

Comment (arquio) [reply hotelx/hotstc/objects/object_Location.graphql]: https://en.wikipedia.org/wiki/ISO_3166-2:US

File: hotelx/hotstc/objects/object_PropertyType.graphql (hunk 2 above)
Comment (mikimartin88) [reply hotelx/hotstc/objects/object_PropertyType.graphql]: PropertyType code is not unique

File: hotelx/hotstc/objects/object_Area.graphql [hunk 4]
```diff
@@ -0,0 +1,6 @@
+type Area {
```
Comment (arquio) [inline hotelx/hotstc/objects/object_Area.graphql]: Change by
#Area(m²) International System of Units
scalar Area

File: hotelx/hotstc/enums/enum_SquareUnitType.graphql [hunk 5]
```diff
@@ -0,0 +1,4 @@
+enum SquareUnitType {
```
Comment (arquio) [inline hotelx/hotstc/enums/enum_SquareUnitType.graphql]: Del

**Pull request merged**